docs/metrics.md

```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics are exposed at `controller-service` on port `9090`
+
+There are different kinds of exporters available: Prometheus, Google Stackdriver, etc. They can can be configured using [observability configuration](../config/config-observability.yaml). 
```

Comment:
sorry I didn't notice this earlier: "can can"

Reply:
ah my bad rather, fixed :+1: